scripts/erc1155-call-script.js

```diff
@@ -0,0 +1,64 @@
+const ERC1155TokensContractaddress = "0x4179DAA9Cf45500FFB75fe720DF043fAbde48D19";
+const Account2 = "0x395a4D90b0F7c34922D09e2652cFCa7362e3a3B4";
+async function fetch() {
```

Comment:
What is this function really doing? Is it like a test?

Reply:
Yes this test function for testnet, Using this function anyone can interact with deployed smart contract.

Reply:
could use better naming than just "fetch" - what about `example_interactions` or something along the lines?
or is this common practice in the solidity world?

Reply:
Yah sure.. will change with `example_interactions` as of now, Kind of common function, its hardhat init example script.

Reply:
Solved